ui/admin/app/components/roles/role/actions/index.hbs

```diff
@@ -3,38 +3,35 @@
   SPDX-License-Identifier: BUSL-1.1
 }}
 
-{{!TODO Refactor page to use hds dropdown in ICU-13879}}
-<Rose::Dropdown
-  @text={{t 'actions.manage'}}
-  @dropdownRight={{true}}
-  as |dropdown|
->
+<Hds::Dropdown as |dd|>
+  <dd.ToggleButton @text={{t 'actions.manage'}} @color='secondary'/>
   {{#if (can 'addPrincipals role' @model)}}
-    <dropdown.link
+    <dd.Interactive
+      @text={{t 'resources.role.principal.actions.add-principals'}}
       @route='scopes.scope.roles.role.add-principals'
-      data-test-manage-dropdown-principals
-    >
-      {{t 'resources.role.principal.actions.add-principals'}}
-    </dropdown.link>
+    />
+    <dd.link
+      @route='scopes.scope.roles.role.add-principals'
+      data-test-manage-dropdown-principals>
+        data-test-manage-dropdown-principals      
+    </dd.link>
```

Comment:
Looks like we have rose and hds code mixed in here. This section of code is rose and should be removed. Just need to add the `data-test` selector to the `<dd.Interactive />` item above.